dev-tools/v2tool/README.md

```diff
@@ -0,0 +1,57 @@
+# V2Tool
+
+`v2tool` is a simple CLI tool to help develop clients for the `elastic-agent` V2 API. The tool acts like a mock V2 server,
+configuring, starting, and stopping the client based on user config.
+
+In it's current state, `v2tool` is fairly simple:
+
+```
+Usage:
+  v2tool -d -c ./input.yml [CLIENT_EXEC] [flags]
+
+Flags:
+  -c, --c string   Configuration file, relative to path.config (default "./v2-config.yml")
+  -d, --d          enable debug-level logging
+  -h, --help       help for v2tool
+```
+
+## Config and rules
+
+To use it, create a `v2-config.yml` file that the `v2tool` will used to configure and start the client.
+The config must have an `args`, `rules` and `inputs` section:
+```
+args:
+  - loadgenerator # The list of CLI args that will be passed to the V2 client application
```

Comment:
It's not clear to me what the `-loadgenerator` args key does here.

Also it might be helpful to use a more well known input as the primary example (Filebeat logs for example) so people can relate it to what they are used to seeing in agent policies.

Reply:
Ah yah, I only really had time to test against one input, in this case, that one CLI arg is because the elastic-agent-inputs run like `./elastic-agent-input [INPUT_NAME]`.